pkg/action/cl002/ac002/executor.go

```diff
@@ -0,0 +1,115 @@
+package ac002
```

Comment:
File which is no boilerplate. Check our created TC and verify if all pods with host network matches with our numbers